src/components/WorkPoolQueueCombobox.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <p-combobox v-model="internalValue" :options="options" :empty-message="emptyMessage">
+    <template #combobox-options-empty>
+      No work queues
```

Comment:
```suggestion
      No work pool queues
```
Not sure if this is correct

Reply:
I don't think it matters as much since we do use "work queue" in the UI, so people can still think of workPoolQueue as workQueues they are so used to 
<img width="566" alt="Screenshot 2023-02-08 at 1 40 01 PM" src="https://user-images.githubusercontent.com/40467112/217622339-e3494ffe-c5b9-416f-bb49-bf8555ff1464.png">